vmdb/app/controllers/application_controller/filter.rb

```diff
@@ -449,17 +450,14 @@ def adv_search_toggle
       if @edit[:adv_search_open] == true
         @edit[:adv_search_open] = false
         page << "$('#adv_search_img').prop('src', '/images/toolbars/squashed-true.png')"
-        page << javascript_hide("adv_search_div")
+        page << javascript_hide("advsearchModal")
```

Comment:
@Fryguy This is a boostrap/patternfly ID. It's the standard style for bootstrap  ids, classes, etc 
"adv_search_div" was our old code.